JarrydWorland/2021_Capstone_GP2_Team
# Fixed code syntax in SceneFaderBehaviour and MenuBehaviour scripts

### Changes include

- Changed SceneFader to SceneFaderBehaviour
- Changed SceneFaderBehaviour reference in MenuBehaviour from public to protected
- Capitalized Image reference in SceneFaderBehaviour

### How to test

- Open Menu scene and select "Play Game" button
- Alternatively, open Gameplay scene, pause the game and select "Exit to Main Menu" button
- Review SceneFaderBehaviour.cs, and the MenuBehaviour scripts for Death, Pause and Main to see syntax changes.
